sdk/identity/identity/src/msal/nodeFlows/msalClient.ts

```diff
@@ -184,12 +195,17 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
       state.cachedAccount = accounts[0];
     }
 
+    // Keep track and reuse the claims we received across challenges
+    if (options.claims) {
+      state.cachedClaims = options.claims;
+    }
+
     // TODO: broker
```

Comment:
curious what this TODO is for

Reply:
Ah yes, there's some logic for nativeBroker that I need to port over still. I updated the comment to link to the changes directly